### PR TITLE
[sqllab] Fix sql lab resolution link

### PR DESCRIPTION
**Pull request by timifasubaa**
This PR makes the error resolution link get passed to the frontend separately in the payload as opposed to preformed html from the backend.

@graceguo-supercat @michellethomas @mistercrunch

### CODE REVIEW

**Review by graceguo-supercat — APPROVED** [review]
This lgtm. But want to see if any other people has comments.

File: superset/assets/src/SqlLab/actions.js [hunk 1]
```diff
@@ -114,7 +114,11 @@ export function fetchQueryResults(query) {
         if (err.responseJSON && err.responseJSON.error) {
           msg = err.responseJSON.error;
         }
-        dispatch(queryFailed(query, msg));
+        let link = '';
```
Comment (john-bodley) [inline superset/assets/src/SqlLab/actions.js]: Can this logic be added to queryFailed so it’s not repeated.

File: superset/assets/src/SqlLab/components/ResultSet.jsx [hunk 2]
```diff
@@ -155,7 +155,11 @@ export default class ResultSet extends React.PureComponent {
     if (query.state === 'stopped') {
       return <Alert bsStyle="warning">Query was stopped</Alert>;
     } else if (query.state === 'failed') {
-      return <Alert bsStyle="danger">{query.errorMessage}</Alert>;
+      return (
+        <Alert bsStyle="danger">
+          {query.errorMessage}
+          {query.link && <a href={query.link}> (Common errors and their resolutions)</a>}
```
Comment (mistercrunch) [inline superset/assets/src/SqlLab/components/ResultSet.jsx]: Let's use a `i18n` string `{t('(Common errors and their resolutions)')}`

**Comment by timifasubaa** [comment]
@johnbodley @mistercrunch done

**Comment by timifasubaa** [comment]
PING

**Review by john-bodley — APPROVED** [review]

**Pull request merged**